src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/PandocCompiler.kt

```diff
@@ -0,0 +1,60 @@
+/**
+    Copyright 2018 BarD Software s.r.o
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Author: Mikhail Shavkunov (@shavkunov)
+ */
+
+package com.bardsoftware.backend.fleet.rmarkdown
+
+import org.slf4j.LoggerFactory
+import java.nio.file.Path
+
+const val PANDOC_DEFAULT_VALUE = "pandoc"
+const val CP_VALUE = "cp"
```

Comment:
This was supposed to be in test code, no?

Reply:
Yes, but I'm not able to use `CP_VALUE` constant in `PandocCompiler` from `PandocTest`

Reply:
You don't need it if the whole command is in the config

Reply:
Yes, I've finally got it :)